components/raftstore/src/store/worker/read.rs

```diff
@@ -208,12 +214,80 @@ where
 }
 
 /// #[RaftstoreCommon]: LocalReadContext combines some LocalReader's fields for temporary usage.
-pub struct LocalReadContext<'a, E>
+pub struct SnapCacheContext<'a, E>
 where
     E: KvEngine,
 {
-    read_id: &'a mut ThreadReadId,
-    snap_cache: &'a mut Box<Option<Arc<E::Snapshot>>>,
+    // read_id: &'a mut ThreadReadId,
```

Comment:
The comments and code are not corresponded

Reply:
Good catch. I forget to update it.